index.js

```diff
@@ -36,6 +36,7 @@ function isTrusted(parsed, trustedHost) {
 
 module.exports = function(opts) {
 	opts = opts || {};
+	opts.getJwt = opts.getJwt || getJwt;
```

Comment:
Doing it like this means we're requiring more than necessary. I would want to see a path where `'frau-jwt'` (unframed) isn't required for the framed case.

Also modifying the input options isn't amazing, since the consumer will see the effect of that.